git-publish

```diff
@@ -553,9 +553,15 @@ def main():
         tag(tag_name_staging(topic), tag_message, annotate=anno, force=True)
 
     prefix = options.prefix
-    if prefix is None:
+    if prefix is not None:
+        # when prefix == "", it will clear per-branch prefix
```

Comment:
Please don't make "" a special value.  None of the other options work like this and the inconsistency is likely to lead to bugs.  This trick isn't discoverable either because you didn't document it.

Also, why strip the prefix?

The rest of this patch looks good.  It's a useful feature.